cfgov/jinja2/v1/_includes/organisms/info-unit-group-2.html

```diff
@@ -0,0 +1,68 @@
+{# ==========================================================================
+
+   Info Unit Group
+
+   ==========================================================================
+
+   Description:
+
+   Create an info unit group organism when given:
+
+   value:                         Object defined from a StreamField block.
+
+   value.heading:                 The heading for the info unit group.
+
+   value.heading_level:           The heading level (H2, H3, or H4).
+
+   value.heading_icon:            An optional cf-icons icon code that
+                                  triggers the insertion of an icon before
+                                  the heading.
+
+   value.intro:                   An introduction for the info unit group.
+
+   value.link_image_and_heading:  Boolean for whether the image and heading
```

Comment:
Did this get copied here by accident? There's no links tuple in the group, only the individual info units